com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Shaders/DoFMinMaxDilate.compute

```diff
@@ -34,7 +34,7 @@ void KMain(uint3 dispatchThreadId : SV_DispatchThreadID)
     float4 tileInfo = _InputTexture[COORD_TEXTURE2D_X(dispatchThreadId.xy)];
 
     int2 startIndex = max((int2)dispatchThreadId.xy - int2(1, 1), int2(0, 0));
-    int2 endIndex =  min((int2)dispatchThreadId.xy + int2(1, 1), _ScreenSize.xy / TILE_RES - int2(1, 1));
+    int2 endIndex =  min((int2)dispatchThreadId.xy + int2(1, 1), _PostProcessScreenSize.xy / TILE_RES - int2(1, 1));
```

Comment:
This is the fix I mentioned to both of you in slack @pmavridis  (cc: @kecho as it is going to fix some issues with DLSS  too) 

(A similar change also in https://github.com/Unity-Technologies/Graphics/pull/5066/commits/c3fc9ff70fc6e887f022869811ffd35b241aae74#diff-615e5f83c56e2e6631a5ef1db8170921d0702eed82a4f08cec8c97f22456f8c5R33 )